### DISCUSSION TITLE
Task 06 - Problem 4: Improved 3D Planar Measurement Tool (Comparing with Task 04 or 05?!)

### DISCUSSION THREAD

**Discussion by hastishwan-uw** [chulminy/CIVE497-CIVE700#170] [Q&A] [answered]
Just to clarify, we didn't include any measurements in Task 05, but in Task 04, specifically Problem 7, we developed the 3D Planar Measurement Software. Could you pinpoint exactly which problem we should be referencing for comparison?

Task 06
![image](https://github.com/chulminy/CIVE497-CIVE700/assets/48458021/22368401-7ec1-4367-b136-1c47a02e4d06)

Task 04
![image](https://github.com/chulminy/CIVE497-CIVE700/assets/48458021/3ae02d04-3cf7-4bb6-8808-c7bd9edb595b)

**Comment by chulminy** [comment, marked as answer]
Hi Hasti

There are two different ways of measuring the distances from the previous task and this task. You can compare the measurement. I think the tasks that you mention are correct. 

Chul Min
